base/tests/unittests/com/google/idea/blaze/base/projectview/ProjectViewSetTest.java

```diff
@@ -120,6 +121,7 @@ public void testProjectViewSetSerializable() {
                         ScalarSection.builder(BuildConfigSection.KEY)
                             .set(new WorkspacePath("test")))
                     .add(ScalarSection.builder(UseExclusionPatternsSection.KEY).set(false))
+                    .add(ScalarSection.builder(ViewProjectRootSection.KEY).set(false))
```

Comment:
Could you add a test where we actually set it in a project view? We don't have to assert anything on the actual behaviour, I just want to check that it parses.